examples/ale_regression_boston.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>Has anyone proved consistency for ALE? What I mean is that if we take two models A and B, and we know feature x is more important in B vs A, then does ALE tell us the same story? I think we should be able to generate some functions of the same variables to check this empirically. This might be a future development and we can talk about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>Yea, this is not really studied at all as far as I can tell, but it's a natural idea to compare different models and I wanted to include the plot to show that you can do it very easily. Have been careful about the interpretation, but as you say would have to spend time on studying this in more detail.</p>